### PR TITLE
Fix vitepress' breaking change

### LINKED ISSUE #43: bug: broken support for vitepress@1.0.0-alpha.38
## Description

Seach bar seems to either unexpectedly disappear, fill console with `cannot read ... of undefined` errors or redirect to blank pages after searching for something.

## Reproduction

https://github.com/Raiondesu/vuse-rx/tree/feature/search
https://vuse-rx-git-feature-search-raiondesu.vercel.app

## Hypothesis

VitePress seems to have accidentally made a breaking change in [this commit](https://github.com/vuejs/vitepress/commit/8de2f4499d9364d85e6070ce4b94651a1902b101), which causes vitepress-plugin-search to misbehave when using a version of VitePress that includes the aforementioned commit.

## Proposed fix

Change
```ts
const { localePath } = useData();
...
origin.value = window.location.origin + localePath.value;
```
to
```ts
const { localeIndex } = useData();
...
origin.value = window.location.origin + localeIndex.value;
```

in `Search.vue`

### PR DESCRIPTION
**Pull request by Raiondesu**
fixes #43

### CODE REVIEW

File: src/Search.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { useData } from "vitepress";
 // @ts-ignore
 import Index from "./module/index.js";
 
-const { localePath } = useData();
+const { localeIndex } = useData();
```
Comment (emersonbottero) [inline src/Search.vue:7]: Could you try to import both and check witch one has data?
Like const path = localePath || localeIndex
This way it won't brake CI from those already using it.

An 
//Todo: delete deprecate code on final version comment would be nice then.

**Review by emersonbottero — CHANGES_REQUESTED** [review]

File: src/Search.vue (hunk 1 above)
Comment (emersonbottero) [reply src/Search.vue:7]: Also please increase the version of the package 🙂

Comment (webpon) [reply src/Search.vue:7]: i also meet this problem

**Comment by Raiondesu** [comment]
Does this package mirror the vitepress version? It seems like it on first glance.

**Pull request merged**